### DISCUSSION TITLE
Can i Contribute?

### DISCUSSION THREAD

**Discussion by sourabh053** [danishzayan/YourQuotes-App#66] [General]
@danishzayan Hi, is this repo active? I would like to contribute. Please respond!

**Comment by danishzayan** [comment]
> @danishzayan Hi, is this repo active? I would like to contribute. Please respond!

Yes this repos is active and you can contribute.

**Reply by sourabh053** [reply]
Okay, then I will be actively working on the pending features.

**Reply by danishzayan** [reply]
> Okay, then I will be actively working on the pending features.

Good...

**Comment by sourabh053** [comment]
@danishzayan , now i will be working on login feature.
